src/MakePotCommand.php

```diff
@@ -163,6 +171,10 @@ public function __invoke( $args, $assoc_args ) {
 			$this->exclude = array_unique( $this->exclude );
 		}
 
+		if ( isset( $assoc_args['headers'] ) ) {
+			$this->headers = json_decode( $assoc_args['headers'], true );
```

Comment:
WP-CLI has a nice little utility method for this so you don't need `isset()` and `json_decode()`.

Example:

```php
$array_arguments = array( 'headers' );
$assoc_args      = \WP_CLI\Utils\parse_shell_arrays( $assoc_args, $array_arguments );
```